src/stock_str.rs

```diff
@@ -332,6 +332,9 @@ pub enum StockMessage {
 
     #[strum(props(fallback = "Not connected"))]
     NotConnected = 121,
+
+    #[strum(props(fallback = "%1$s changed his/her address from %2$s to %3$s"))]
```

Comment:
+1 for "changed their address" - other messengers use that too in such situations, they/them is a gender neutral english pronoun mainly used if you don't know someone's gender. It is nowadays also used as a nonbinary pronoun, but the far older usage is as a gender neutral pronoun. they/them would be the correct grammar in this situation.

It might not be super easy to translate into languages which don't have gender neutral pronouns; I like to use "deren" in german, but it's not super common. I would leave that to the translators, they can think about this much better than we can.

Reply:
Translations happen a bit randomly, though.  If we can avoid the issue it's easiest also for translators.  but i have no concrete suggestion atm.

Reply:
I mean, "Alice's address changed" would be possible, but then we don't communicate that changing her address was a deliberate action taken by Alice.